samples/graalvm-native-jna/src/main/resources/META-INF/native-image/proxy-config.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+    "interfaces":["com.example.JnaNative$CLibrary"]
+  }
+]
```

Comment:
Full required configuration for the native image sample (pending reachability handler question)

Reply:
we don't even need that anymore